packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -89,6 +90,16 @@ const RootStack: React.FC = () => {
       } catch {
         try {
           // Try connectionless here
+          const queryParams = parseUrl(deepLink).query
+          const param = queryParams['d_m'] ?? queryParams['c_i']
```

Comment:
`oob` is also another [known query string parameter](https://github.com/hyperledger/aries-framework-javascript/blob/a0458febe4902e823c81b8e394a39b6bdbff31bc/packages/core/src/utils/parseInvitation.ts#L78)

Reply:
that type will be handled above on line 85